Sources/Docopt.swift

```diff
@@ -18,7 +18,7 @@ open class Docopt : NSObject {
     fileprivate let optionsFirst: Bool
     fileprivate let arguments: [String]
     
-    open static func parse(_ doc: String, argv: [String], help: Bool = false, version: String? = nil, optionsFirst: Bool = false) -> [String: AnyObject] {
+    @objc open static func parse(_ doc: String, argv: [String], help: Bool = false, version: String? = nil, optionsFirst: Bool = false) -> [String: AnyObject] {
```

Comment:
swift 4 requires more explicit marking of objc compatibility. without this, the parse function is invisible to objc even though the class itself it visible.